trunk/workflow-manager/src/test/protractor/pageObjects/jobs.pageObject.js

```diff
@@ -33,8 +33,7 @@ var JobsPage = function() {
 
     this.pageName = "Job Status";
     this.currentJob = null;
-
-    this.video_uri = "http://somehost-mpf-4.mitre.org/rsrc/datasets/samples/face/new_face_video.avi";
+    this.video_uri = "https://github.com/openmpf/openmpf/blob/master/trunk/mpf-system-tests/src/test/resources/samples/face/new_face_video.avi?raw=true";
```

Comment:
For consistency with `EXT_IMG` this should be: https://raw.githubusercontent.com/openmpf/openmpf/develop/trunk/mpf-system-tests/src/test/resources/samples/face/new_face_video.avi